app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -209,10 +209,15 @@ class CtaViewModel @Inject constructor(
                     }
                 }
             }
+
             // Is Serp
-            if (!daxDialogSerpShown() && isSerpUrl(it.url)) {
-                return DaxDialogCta.DaxSerpCta(onboardingStore, appInstallStore)
+            if (isSerpUrl(it.url)) {
```

Comment:
Nice work making this self-documenting. Let's delete the `// Is Serp` comment, it's no longer needed.